technic_worldgen/rubber.lua

```diff
@@ -92,7 +128,7 @@ if technic.config:get_bool("enable_rubber_tree_generation") then
 				y = (maxp.y - minp.y) / 2 + minp.y,
 				z = (maxp.z - minp.z) / 2 + minp.z}
 		local pos = minetest.find_node_near(tmp, maxp.x - minp.x,
-				{"default:dirt_with_grass"})
+				{has_mcl and "mcl_core:dirt_with_grass" or "default:dirt_with_grass"})
```

Comment:
You can just put both nodes in the list here.
```suggestion
				{"default:dirt_with_grass", "mcl_core:dirt_with_grass"})
```

Reply:
This was actually like this before but suggested that it might be better to use only nodes that are actually registered.

Reply:
Oh, I missed that. Does it actually make a difference though?

Reply:
Probably not really very significant performance wise, it just doubles the size of the underlying search loop.

Reply:
> EDIT: I checked the engine code, and the nonexistent nodes are ignored before the searching, so there shouldn't be any impact on performance.

Seems like that, basically only adding minimal overhead for api, group substring check and content id search.
Actual optimization, which would still be insignificant, would be to cache and reuse same table.

Anyway, my reasoning in original change request was about avoiding things that clearly does not exist. Isn't really very important here but something to consider: should it be used if you know for sure it does not exist?

Reply:
> I checked the engine code, and the nonexistent nodes are ignored before the searching, so there shouldn't be any impact on performance.
changed again

Reply:
> Anyway, my reasoning in original change request was about avoiding things that clearly does not exist. Isn't really very important here but something to consider: should it be used if you know for sure it does not exist?

I preffer this method, changed again